docs/_docs/reference/other-new-features/export.md

```diff
@@ -77,6 +77,18 @@ A member is _eligible_ if all of the following holds:
 
 It is a compile-time error if a simple or renaming selector does not identify any eligible members.
 
+The qualifier expression `qual` can contain selections as well as applications
+to arguments. However, if a type member is exported, `qual` must be a stable path.
+
+Example:
+
+```scala
+class C(x: Int) { type T; def m = x }
+export C(2).m  // OK, generates:   def m = C(2).m
```

Comment:
Why that desugaring over creating a synthetic val (or lazy val?) for the application and then a `def m` that uses that single value?

Reply:
That would not work for the primary use case: exports in extension methods. Extension methods only accept defs.

Reply:
What if `def m : T = ???` and we `export C(2).m`, or if the method has arguments that depend on `T`?

Reply:
Are internal classes not considered stable? Because this currently fails:
```scala
class C(x: Int) { class Internal}
export C(2).Internal
```

Reply:
No, it's a type, this is as expected.